### PR TITLE
Adds some new, cool-looking bottles to the Glass Recycler

### PR DESCRIPTION
**Pull request by genessee596**
[FEATURE]

## About the PR 

This PR adds five new bottles, printable at a Glass Recycler. They are functionally identical to a drinking bottle, but with different sprites, names, and descriptions. They all cost 1 glass-point(?) to print, and all hold 50 units. This does not change or remove anything else that the Glass Recycler can make.

image
From left to right; 'Long Bottle', 'Tall Bottle', 'Rectangular Bottle', 'Square Bottle', 'Wide Bottle'

**I DID NOT SPRITE THESE BOTTLES, AND TAKE ZERO CREDIT FOR THE SPRITES.** They're used as bases for various drinks you can get from vendors, but these all lack a label and start off empty. The sprites are already in the game files, hence why this PR does not add or change any .dmi files. (Woohoo, no icon conflicts!)

## Why's this needed? 

I like bottles. I really, really like bottles; in real life, I have a bottle collection. When new bottle sprites came to town, I was hyped; and for the most part, the new bottles looked great. Except for the one that the bartender could make, anyways - that one just looked like a crappy soda-bottle codersprite to me. Originally, this PR was going to just add the old bottle sprite back alongside the new bottle sprite as seperate items, but I saw that cool-ass blank bottle sprites and took my opportunity.

These bottles will let creative barmen spruce up their creations, and look great on shelves. They're also useful for organizing your drinks if you have a lot that look the same color. And they're pretty as hell, so that's a plus.

TL;DR: current soda bottle for barman bad, new sprites for barman bottle good

EDIT: learned to english
EDIT 2: 'Masculine Bottles' renamed to 'Wide Bottles'. Bottle fetishists worldwide howl in sorrow.

## Changelog

```
(u)Genessee:
(+)Five new types of bottle can now be printed at a Glass Recycler.
```

### CODE REVIEW

**Comment by genessee596** [comment]
First Goonstation PR, so please have mercy

many thanks

**Comment by awfulworldkid** [comment]
please do not actually add an item named "Masculine Bottle", i'm begging you, why are you like this

**Comment by genessee596** [comment]
> please do not actually add an item named "Masculine Bottle", i'm begging you, why are you like this

listen i couldn't think of a good adjective for the tequila bottle and it kind of looks like a man okay
im just a victim of circumstance i swear

**Comment by JohnBigless** [comment]
> > please do not actually add an item named "Masculine Bottle", i'm begging you, why are you like this
> 
> listen i couldn't think of a good adjective for the tequila bottle and it kind of looks like a man okay
> im just a victim of circumstance i swear

Strengthened Bottle? Shouldered Bottle? *Stronk Bottle?*

**Comment by genessee596** [comment]
> > > please do not actually add an item named "Masculine Bottle", i'm begging you, why are you like this
> > 
> > 
> > listen i couldn't think of a good adjective for the tequila bottle and it kind of looks like a man okay
> > im just a victim of circumstance i swear
> 
> Strengthened Bottle? Shouldered Bottle? _Stronk Bottle?_

bottle of unending strength and power

**Comment by Adharainspace** [comment]
my masculine bottle is too strong for you, traveller

**Comment by TDHooligan** [comment]
would love this for both fancy drinks and tonics, the name 'wide-shoulder bottle' could work

**Comment by genessee596** [comment]
> would love this for both fancy drinks and tonics, the name 'wide-shoulder bottle' could work

'Wide Bottle'! That actually works way better than 'Masculine Bottle'. Thank you for the suggestion!
